browser/src/map/handler/Map.Keyboard.js

```diff
@@ -594,18 +591,9 @@ L.Map.Keyboard = L.Handler.extend({
 		else if (ev.type === 'keydown') {
 			var key = ev.keyCode;
 			var map = this._map;
-			if (key in this._panKeys && !ev.shiftKey) {
-				if (map._panAnim && map._panAnim._inProgress) {
-					return;
-				}
-				map.fire('scrollby', {x: this._panKeys[key][0], y: this._panKeys[key][1]});
-			}
-			else if (key in this._panKeys && ev.shiftKey &&
-					!docLayer._textCSelections.empty()) {
-				// if there is a selection and the user wants to modify it
-				if (keyEventFn) {
-					keyEventFn('input', charCode, unoKeyCode);
-				}
+			if (['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown', 'Home', 'End'].includes(ev.code)) {
```

Comment:
I think `_panKeys` is not used at all? (no assignments in the code I can see).

Reply:
Yes a forgotten local variable. I cleaned it.